app/grandchallenge/components/templates/components/civ_set_delete_confirm.html

```diff
@@ -13,7 +13,11 @@
 {% block content %}
     <h1>Delete {{ base_object.title }} {{ base_object.civ_set_model|verbose_name_plural|title }}</h1>
     {% if delete_count == 0 %}
-        <p>No editable {{ base_object.civ_set_model|verbose_name_plural }} selected.</p>
+        <p>No editable {{ base_object.civ_set_model|verbose_name_plural }} selected.
+        {% if base_object.civ_set_model|verbose_name_plural == "display sets" %}

```

Comment:
The verbose name should not be used for matching as that is easily changed. Does this error message even make sense? How does the user get here? Shouldn't the delete all button be disabled if it cannot be used?

Reply:
The original issue is that it is not clear what needs to be done to delete all display sets (after clicking the delete all button and seeing the message "No editable display sets are selected". 
I think disabling the delete all button while also the display sets are non-selectable would make it even more vague to the user on what should be done.

Reply:
Why not disable the button and add a tooltip telling them what they need to do to be able to click the button? https://getbootstrap.com/docs/4.1/components/tooltips/#disabled-elements

Reply:
Done